docs/example.yaml

```diff
@@ -16,6 +16,13 @@ spec:
       kind: StatefulSet
   version: "4.9.4"
   description: "WordPress is open source software you can use to create a beautiful website, blog, or app."
+  icons:
+    - src: "https://s.w.org/style/images/about/WordPress-logotype-wmark.png"
+      type: "image/png"
+      size: "1000x1000"
+    - src: "https://s.w.org/style/images/about/WordPress-logotype-standard.png"
```

Comment:
Is this something we can legally use?

Reply:
We are referencing a URL rather than using the image. If someone uses the example it is upon them to make sure they follow the policy when they use the images. Is there a problem with referencing these URLs? Something I'm not aware of.